yb-voyager/cmd/exportSchema.go

```diff
@@ -260,6 +265,12 @@ func runAssessMigrationCmdBeforExportSchemaIfRequired(exportSchemaCmd *cobra.Com
 		}
 	}
 
+	if !isSourcePasswordSetViaFlag {
```

Comment:
why not always pass it? why do we need to check? 
In fact, can we set it in the ENV var just to be on the safe side?

Reply:
yeah, that we can do. I thought we can explicitly set for the but I like the idea of running it through ENV in all cases as its an internal command we are running.